schema/common/validation/epiphany-cluster.yml

```diff
@@ -79,19 +79,24 @@ properties:
         type: object
         title: The Credentials Schema
         required:
-        - key
-        - secret
+        - access_key_id
+        - secret_access_key
         properties:
-          key:
+          access_key_id:
             "$id": "#/properties/specification/properties/cloud/properties/credentials/properties/key"
             type: string
             title: The Key Schema
             pattern: "^(.*)$"
-          secret:
+          secret_access_key:
             "$id": "#/properties/specification/properties/cloud/properties/credentials/properties/secret"
             type: string
             title: The Secret Schema
             pattern: "^(.*)$"
+          session_token:
+            "$id": "#/properties/specification/properties/cloud/properties/credentials/properties/token"
+            type: string
+            title: The session token
+            pattern: "^(.*)$"
```

Comment:
How about changing this pattern to not allow spaces?
Then we could avoid `credentials.session_token.strip()` in `APIProxy.py`.

https://stackoverflow.com/questions/41917270/use-regex-in-json-schema-to-validate-a-string-for-no-spaces

Reply:
@atsikham ?